app/assets/javascripts/jquery.persistentModal.js

```diff
@@ -82,8 +82,7 @@
 
       function createModal() {
         var jqxhr = $.ajax({
-          url: linkTarget,
-          dataType: 'script'
```

Comment:
this is different too ...

Reply:
Yes, it's asking for HTML, but it was saying "script" which would attempt to eval the results as javascript.